src/cryptoadvance/specter/server_endpoints/wallets.py

```diff
@@ -1538,3 +1610,43 @@ def addresses_list_to_csv(wallet):
         yield data.getvalue()
         data.seek(0)
         data.truncate(0)
+
+
+# wallet addresses list to user-friendly CSV format
+def wallet_addresses_list_to_csv(addresses_list):
```

Comment:
Please put the documentation below the function name like:
```suggestion
def wallet_addresses_list_to_csv(addresses_list):
''' wallet addresses list to user-friendly CSV format'''
```
This way, your IDE will give you a description of that function. Maybe you can also add docs to the parameter to answer, what exactly an address_list is suppose to be.

Reply:
Ok. I improved the documentation of the function.